Module not found: Error: Default condition should be last one

I'm trying to use this package in an Angular app and I'm encountering the following error:
> Module not found: Error: Default condition should be last one

After doing some research online, I found this solution: 
https://stackoverflow.com/a/76127619

I believe the issue comes from this commit (7f4e871). It seems like `types` should be placed before `default`:
https://github.com/Touffy/dl-stream/blob/7f4e871f594f4724fff30c77cc61e8c2241a4692/package.json#L6-L9

--- a/package.json
+++ b/package.json
@@ -4,8 +4,8 @@
   "description": "A specialized WHATWG Stream for seamlessly queueing many downloads",
   "browser": "index.js",
   "exports": {
-    "default": "./index.js",
-    "types": "./index.d.ts"
+    "types": "./index.d.ts",
+    "default": "./index.js"
   },
   "types": "index.d.ts",
   "type": "module",

